docs/fastdds/property_policies/non_consolidated_qos.rst

```diff
@@ -11,11 +11,63 @@ Before consolidating a new QoS Policy, it is usually set using this generic QoS
+
+Configuring the DataWriters on ``pull mode`` offers an alternative by letting each reader pace its own data stream.
+It works by notifying the reader what it is missing, and wait for it to request only as much as it can handle.
+At the cost of greater latency, this model can deliver reliability while using far fewer packets than in ``push mode``.
+
+DataWriters periodically announce the state of their queue by means of a heartbeat.
+Upon reception of the heartbeat, DataReaders will request the DataWriter to send the samples they want to process.
+Consequently, the publishing rate can be tuned setting the heartbeat period accordingly.
+
+.. list-table::
+   :header-rows: 1
+   :align: left
+
+   * - PropertyPolicyQos name
+     - PropertyPolicyQos value
+     - Default value
+   * - ``"fastdds.push_mode"``
+     - ``"true"``/``"false"``
+     - ``"true"``
+
++----------------------------------------------------------------------------------------------------------------------+
+| **C++**                                                                                                              |
++----------------------------------------------------------------------------------------------------------------------+
+| .. literalinclude:: /../code/DDSCodeTester.cpp                                                                       |
+|    :language: c++                                                                                                    |
+|    :start-after: // PULL_MODE_DATAWRITER                                                                             |
+|    :end-before: //!--                                                                                                |
+|    :dedent: 8                                                                                                        |
++----------------------------------------------------------------------------------------------------------------------+
+| **XML**                                                                                                              |
++----------------------------------------------------------------------------------------------------------------------+
+| .. literalinclude:: /../code/XMLTester.xml                                                                           |
+|    :language: xml                                                                                                    |
+|    :start-after: <!-->PULL_MODE_DATAWRITER<-->                                                                       |
+|    :end-before: <!--><-->                                                                                            |
++----------------------------------------------------------------------------------------------------------------------+
+
+.. note::
+    Communication to readers running on the same process will always use ``push mode``
```

Comment:
```suggestion
    Communication to readers running on the same process (:ref:`intraprocess-delivery`) will always use ``push mode``
```

I suggest adding the reference in case the user wants to know more.